helm/qontract-reconcile/templates/template.yaml

```diff
@@ -783,6 +810,15 @@ parameters:
 - name: PUSHGATEWAY_SECRET_NAME
   value: push-gateway-auth-url
 {{- range $i, $integration := .Values.integrations }}
+{{- $logs := $integration.logs | default dict -}}
+{{- if $logs.govslack_error_filter }}
+- name: GOVSLACK_CHANNEL_ERRORS_ONLY
+  value: "sd-app-sre-govslack-errors-stage"
+  required: false
+{{- break }}
+{{- end }}
+{{- end }}
+{{- range $i, $integration := .Values.integrations }}
```

Comment:
I think it would make sense if we removed the logic around this property definition. I would suggest putting it next to where

```
- name: SLACK_CHANNEL
  value: "sd-app-sre-reconcile-stage"
- name: SLACK_CHANNEL_TRIGGER
  value: "sd-app-sre-triggers-stage"
  ```
  
  is defined.

Reply:
I also note that the google-chat properties were wrapped in an if statement. That made sense because there are sub-properties within that if-statement pulling from an expected k8s secret (i.e. `secretKeyRef`), and there's a whole custom google-chat fluentd config written within it.

But I don't think it's necessary here to have an integration-property "blocking" the definition of `SLACK_RECONCILE_ERR_CHANNEL_NAME`.